mne/filter.py

```diff
@@ -573,6 +574,130 @@ def _check_method(method, iir_params, extra_types):
 
 
 @verbose
+def filter_(data, sfreq, l_freq, h_freq, picks=None, filter_length='10s',
+            l_trans_bandwidth=0.5, h_trans_bandwidth=0.5, n_jobs=1,
+            method='fft', iir_params=None, copy=True, verbose=None):
+    """Filter a subset of channels.
+
+    Applies a zero-phase low-pass, high-pass, band-pass, or band-stop
+    filter to the channels selected by ``picks``.
+
+    ``l_freq`` and ``h_freq`` are the frequencies below which and above
+    which, respectively, to filter out of the data. Thus the uses are:
+
+        * ``l_freq < h_freq``: band-pass filter
+        * ``l_freq > h_freq``: band-stop filter
+        * ``l_freq is not None and h_freq is None``: high-pass filter
+        * ``l_freq is None and h_freq is not None``: low-pass filter
+
+    .. note:: If n_jobs > 1, more memory is required as
+              ``len(picks) * n_times`` additional time points need to
+              be temporaily stored in memory.
+
+    Parameters
+    ----------
+    data : ndarray, shape (n_channels, n_times)
+        The data to filter.
+    sfreq : float
+        The sample frequency in Hz.
+    l_freq : float | None
+        Low cut-off frequency in Hz. If None the data are only low-passed.
+    h_freq : float | None
+        High cut-off frequency in Hz. If None the data are only
+        high-passed.
+    picks : array-like of int | None
+        Indices of channels to filter. If None all channels will be
+        filtered.
+    filter_length : str (Default: '10s') | int | None
+        Length of the filter to use. If None or "len(x) < filter_length",
+        the filter length used is len(x). Otherwise, if int, overlap-add
+        filtering with a filter of the specified length in samples) is
+        used (faster for long signals). If str, a human-readable time in
+        units of "s" or "ms" (e.g., "10s" or "5500ms") will be converted
+        to the shortest power-of-two length at least that duration.
+        Not used for 'iir' filters.
+    l_trans_bandwidth : float
+        Width of the transition band at the low cut-off frequency in Hz
+        (high pass or cutoff 1 in bandpass). Not used if 'order' is
+        specified in iir_params.
+    h_trans_bandwidth : float
+        Width of the transition band at the high cut-off frequency in Hz
+        (low pass or cutoff 2 in bandpass). Not used if 'order' is
+        specified in iir_params.
+    n_jobs : int | str
+        Number of jobs to run in parallel. Can be 'cuda' if scikits.cuda
+        is installed properly, CUDA is initialized, and method='fft'.
```

Comment:
Not AFAIK because you can't do it in the frequency domain, and you can't otherwise parallelize it that well